### PR TITLE
Add configuration reference

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
++ grep -vE (^vt=|^console-provider=vt|^display-layout=) /var/snap/ubuntu-core-kiosk/117/ubuntu_core_kiosk.config
++ exec /snap/ubuntu-core-kiosk/117/bin/run-kiosk /snap/ubuntu-core-kiosk/117/usr/local/bin/ubuntu_core_kiosk --help
++ dirname /run/user/1000/snap.ubuntu-core-kiosk
++ XDG_RUNTIME_DIR=/run/user/1000
++ export XDG_RUNTIME_DIR
++ mkdir -p /run/user/1000 -m 700
++ [ -n wayland-0 ]
++ [ -e /run/user/1000/wayland-0 ]
++ unset WAYLAND_DISPLAY
++ exec /snap/ubuntu-core-kiosk/117/usr/local/bin/ubuntu_core_kiosk --help
```
Comment (wmww) [inline REFERENCE.md]: Can you drop all this bash garbage? Even if it's what gets printed out by the actual command, I don't think it needs to be here.

File: REFERENCE.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+  --wallpaper-top arg (=0x7f7f7f)       Colour of wallpaper RGB
+  --wallpaper-bottom arg (=0x1f1f1f)    Colour of wallpaper RGB
+  --window-management-trace             log trace message
+  --keymap arg (=gb)                    keymap <layout>[+<variant>[+<options>]]
+                                        , e,g, "gb" or "cz+qwerty" or 
+                                        "de++compose:caps"
+  -h [ --help ]                         this help text
+```
+
+You might notice that the options are described as "command-line options", and they can be supplied that way when running the snap from the command-line (and not as a daemon). That can be a convenient way of testing their effect.
```
Comment (wmww) [inline REFERENCE.md]: Perhaps add an example of doing this?

File: RUNNING_ON_YOUR_DESKTOP.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Ubuntu Core Kiosk
+
+**Disclaimer:** *This isn't about how to deploy on "Classic" linux systems (i.e. not "Ubuntu Core") such as Ubuntu Desktop or Ubuntu Server. For these systems see the `daemon` configuration option in [REFERENCE.md](REFERENCE.md)*
```
Comment (wmww) [inline RUNNING_ON_YOUR_DESKTOP.md]: > **Disclaimer:** *This isn't about how to deploy on "Classic" linux systems (i.e. not "Ubuntu Core")

One too many levels of negatives for me. "this is how to deploy to Ubuntu Core, not classic systems such as Ubuntu Desktop.." sounds better.

Or maybe you you meant this is for classic systems? It seems to me like you are talking about classic and the daemon is for core.

**Review by wmww — CHANGES_REQUESTED** [review]
This whole situation seems to be a bit of a mess (what with sending config files as command line options and all), but that isn't our fault and certainly isn't these doc's fault, which are trying their best.

**Comment by AlanGriffiths** [comment]
Actually, I'm going to merge this so that I can use "permanent" URLs to the docs. Feel free to suggest further improvements.

bors merge

**Pull request merged**